providers/rackspace-cloudservers-us/src/test/java/org/jclouds/rackspace/cloudservers/us/compute/CloudServersUSTemplateBuilderLiveTest.java

```diff
@@ -39,11 +39,11 @@ public CloudServersUSTemplateBuilderLiveTest() {
 
    @Test
    public void testTemplateBuilder() {
-      Template defaultTemplate = this.view.getComputeService().templateBuilder().build();
+      Template defaultTemplate = this.view.getComputeService().templateBuilder().imageNameMatches(".*Ubuntu 14\\.10.*").build();
```

Comment:
This is not correct. The test must check the **default** template. That is, what happens when users **don't provide any** criteria in the template builder. This line **must** be reverted to the original one.

The right fix is to configure which is the default template, and that is done in the ProviderMetadata, by setting the default template property with the right criteria. You can take a look at the [CloudSigma2ApiMetadata](https://github.com/jclouds/jclouds-labs/blob/master/cloudsigma2/src/main/java/org/jclouds/cloudsigma2/CloudSigma2ApiMetadata.java#L62) for an example.